net-vpn/logmein-hamachi-2.1.0.198: version bump

nonno-cicala opened the pull request:
Version bump, cleanup, pruned unfetchable version, LICENSE update
I wrote a wrong bug number in the commit message, sorry, it actually:

Closes: https://bugs.gentoo.org/640604
Closes: https://bugs.gentoo.org/661688

nonno-cicala commented:
One of the 2 bug numbers I wrote is wrong, sorry, it actually

Closes: https://bugs.gentoo.org/640604

monsieurp commented on licenses/LogMeIn:
```diff
@@ -1,647 +1,406 @@
-experiencing the difficulty shall give the other prompt written notice,
-with full details following the occurrence of the cause relied upon.
-Copyright © 2003 – 2009 LogMeIn, Inc.  All rights reserved.
+************ TTeerrmmss ooff SSeerrvviiccee ffoorr LLooggMMeeIInn aanndd GGooTToo SSeerrvviicceess ************
```
Can you fix those funny characters?

nonno-cicala commented:
Actually I used git rm to remove a version that can't be fetched any more ( Bug: https://bugs.gentoo.org/640604 ) and later added the new version, leaving the 2 only releases that I know to work. Git detected this as a rename. Next time I'll do a separate commit for pruning versions that cannot be installed.

I fixed the funny character, whatever I used to read the file interpreted them and I didn't realize they were there, sorry.

Thanks.

nonno-cicala commented on licenses/LogMeIn:
```diff
@@ -1,647 +1,408 @@
-experiencing the difficulty shall give the other prompt written notice,
-with full details following the occurrence of the cause relied upon.
-Copyright © 2003 – 2009 LogMeIn, Inc.  All rights reserved.
+To read the latest LogMeIn Terms and Conditions, visit https://www.logmeininc.com/legal/terms-and-conditions
```
That's what I did, so I added the sentence with the date of the copy.
The previous available release is distributed with its own LICENSE file, so I added it in the licenses as LogMeIn-2013.

Thanks.

nonno-cicala commented:
I think I did everything as requested, tell me if it's fine.
Thanks.

nonno-cicala commented:
I think I did everything as requested, tell me if there's something more to do.
Thanks.

mgorny commented on net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.198.ebuild:
```diff
@@ -0,0 +1,64 @@
+
+src_unpack() {
+	unpack ${A}
+	mv "${P}-$(use x86 && echo x86 || echo x64)" "${S}" || die
```
Look into using `usex` here.

nonno-cicala replied:
OK, I will.

I added the irrelevant changes to make repoman happy. Since they aren't important I didn't commit them alone, is it wrong? What would be the preferred way to act?

I didn't rename the old license, they are actually both new. I left the latest with the old name because I think it will probably stay, while the older could be removed when the older package using it won't be available any more on the servers.

nonno-cicala commented:
So, I put the prune in it's own commit, alone.

And I squashed the irrelevant changes with the LICENSE changes and the version bump with `usex`. Since the version bump requires the LICENSE changes and I had to modify the ebuilds anyway I added the irrelevant changes there. Seems cleaner to me.

May I make it better in some way?

Thanks.
